util/update/run.go

```diff
@@ -49,5 +52,27 @@ func Run(path string) error {
 	// update the model
 	update := newUpdate()
 
+	fmt.Printf("Updated to version: %s", getCurrentVersion(path))
```

Comment:
It seems more efficient to call `getCurrentVersion()` only once after the update, storing the value for use by both of the lines that need it. Especially since it doesn't cache the value in `getCurrentVersion()` itself.

Reply:
You're totally right, updated the PR.